### PR TITLE
Added url as optional parameter in graphql query.

### LINKED ISSUE #189: Articles query should optionally take in a url(string) as an argument
The `articles` graphql query should optionally also be able to take in a url(client website url) as an argument. which would return a list of articles associated to a particular url. without this the `suggested articles` tab wont work

### PR DESCRIPTION
**Pull request by sudeeptarlekar**
This fixes #189 

@breezyboa _a please review

### CODE REVIEW

**Comment by vipulnsward** [comment]
@sudeeptarlekar _a Please add tests for this

**Comment by sudeeptarlekar** [comment]
@breezyboa _a PR is ready to review

File: app/graphql/resolvers/articles_search.rb [hunk 1]
```diff
@@ -4,10 +4,17 @@ class Resolvers::ArticlesSearch < GraphQL::Function
...
   def call(obj, args, context)
     if args[:id].present?
-      Article.where(id: args[:id]).for_organization(context[:organization])
+      if args[:url].present?
+        Url.find_by(url: args[:url]).articles.where(id: args[:id]).for_organization(context[:organization])
+      else
+        Article.where(id: args[:id]).for_organization(context[:organization])
+      end
+    elsif args[:url].present?
+      Url.find_by(url: args[:url]).articles.for_organization(context[:organization])
```
Comment (vipulnsward) [inline app/graphql/resolvers/articles_search.rb]: Move all of this to model and add a test case for this.

**Comment by breezyboa** [comment]
@sudeeptarlekar _a I have made relevant elm side changes can you review once

**Comment by sudeeptarlekar** [comment]
@vipulnsward _a please review

**Pull request merged**